src/lib.rs

```diff
@@ -144,13 +145,18 @@ pub async fn run_cargo_command(
                     "debug"
                 });
 
-            for PackageComponentMetadata { package, .. } in packages {
+            for PackageComponentMetadata { package, metadata } in packages {
+                let metadata = match metadata {
+                    Some(metadata) => metadata,
+                    None => continue,
```

Comment:
This appears to be slightly different logic, skipping packages that don't have metadata, rather than proceeding to try to create components with them. Is that intended?

Reply:
That's intended, as it shouldn't attempt to componentize the build output if `Cargo.toml` is devoid of a component section in the package metadata.